wagtailflags/templates/wagtailflags/flags/create_flag.html

```diff
@@ -3,33 +3,37 @@
 {% block titletag %}Create a flag{% endblock %}
 
 {% block content %}
-    {% include "wagtailadmin/shared/header.html" with title=flag.name icon="tag" %}
-
-    <h2 class="nice-padding">
-        Create a flag
-    </h2>
+    {% trans "Create a flag" as create_flags_str %}
+    {% include "wagtailadmin/shared/header.html" with title=create_flags_str icon="flag" %}
 
     <form class="nice-padding" method="POST" action="{% url 'wagtailflags:create_flag' %}">
         {% csrf_token %}
         <ul class="fields">
-            <li class="actions">
-                {% for field in form %}
-                    {% if field.errors %}
-                        <div class="help-block help-critical" style="margin-top:0;">
-                            {{ field.errors }}
-                        </div>
-                    {% endif %}
-                    <label>{{ field.label }}</label>
-                    <div class="field-content">
-                        <div class="input">
-                            {{ field }}
-                            <span></span>
+            {% for field in form %}
```

Comment:
This introduces a lot of duplication of Wagtail's [panel.html](https://github.com/wagtail/wagtail/blob/v4.0/wagtail/admin/templates/wagtailadmin/shared/panel.html) introduced in 4.0. I tried this and it seems to work reasonably:

```html
{% for field in form %}
    {% include "wagtailadmin/shared/field.html" with field=field %}
{% endfor %}}
```

Reply:
Done in [684ee68](https://github.com/cfpb/wagtail-flags/pull/72/commits/684ee6857d85a03041d2e8b0a7fc8a695d40c852). This is a great discovery that makes me very happy 😁